packages/picasso/src/ShowMore/ShowMore.tsx

```diff
@@ -58,8 +70,11 @@ export const ShowMore = forwardRef<HTMLSpanElement, Props>(function ShowMore(
         color='dark-grey'
         className={className}
         style={style}
+        data-testid={testIds?.contentWrapper}
       >
-        <Truncate lines={!shownMore && rows}>{text}</Truncate>
+        {showContent && (
+          <Truncate lines={!shownMore && rows}>{content}</Truncate>
```

Comment:
```suggestion
          <Truncate lines={rows}>{content}</Truncate>
```

since we have `shownMore` in condition, this will never be used

Reply:
We have `shownMore` in condition for `isContentShown` but that will also be true if rows > 0 regardless of `shownMore`